src/components/common/WalletOverview/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Typography } from '@mui/material'
+import { Suspense } from 'react'
+import type { ReactElement } from 'react'
+
+import EthHashInfo from '@/components/common/EthHashInfo'
+import WalletIcon from '@/components/common/WalletIcon'
+import type { ConnectedWallet } from '@/hooks/wallets/useOnboard'
+import { useAppSelector } from '@/store'
+import { selectChainById } from '@/store/chainsSlice'
+
+import css from './styles.module.css'
+import { isSocialLoginWallet } from '@/services/mpc/module'
+import SocialLoginInfo from '@/components/common/SocialLoginInfo'
+
+export const UNKNOWN_CHAIN_NAME = 'Unknown'
+
+const WalletOverview = ({ wallet }: { wallet: ConnectedWallet }): ReactElement => {
```

Comment:
This is just what `WalletInfo` was before. Didn't touch any of the code in this file and its styles.